### PR TITLE
Publish all historical time series points

### PR DESCRIPTION
**Pull request by BjarkeMeier**
## References

* #616

### CODE REVIEW

File: source/databricks/package/timeseries_publisher.py [hunk 1]
```diff
@@ -22,118 +22,42 @@
...
+         col(Colname.metering_point_id),
+         col(Colname.transaction_id),
+         col(Colname.quantity),
+         col(Colname.quality),
+         col(Colname.time),
+         col(Colname.resolution),
+         col(Colname.year),
+         col(Colname.month),
+         col(Colname.day),
+         col(Colname.registration_date_time)
```
Comment (ZavezX) [inline source/databricks/package/timeseries_publisher.py:41]: Didn't we talk about using the `created_date_time` instead? Or perhaps the `encued_date_time` from the eventhub message?

**Review by ZavezX — CHANGES_REQUESTED** [review]

File: source/databricks/package/timeseries_publisher.py (hunk 1 above)
Comment (BjarkeMeier) [reply source/databricks/package/timeseries_publisher.py:41]: According to the talk yesterday and settlement with PO and SME today it'll be EventHub enqueued time. I'm working on it.

Comment (BjarkeMeier) [reply source/databricks/package/timeseries_publisher.py]: Fixed. @ZavezX and @johevemi would you mind reviewing again?

**Review by ZavezX — APPROVED** [review]
Just some flake8 errors, otherwise it looks fine.

**Pull request merged**